src/styles/content/digits.css

```diff
@@ -0,0 +1,30 @@
+
+
+.digits__table {
+    margin: auto;
+    width: 1100px;
+    padding: 0 0 100px 0
+}
+td {
```

Comment:
BEM restricts the usage of the tag selectors. Use the class instead

Reply:
done